src/blr/config.schema.yaml

```
@@ -52,4 +52,9 @@ properties:
   max_molecules_per_bc:
     type: integer
     description: Max number of molecules allowed for a single barcode (removes if bc has > 260 molecules).
-
+  variant_caller:
+    type: string
```

```suggestion
    type: string
    pattern: "gatk|bcftools|freebayes|deepvariant"
```
Add a pattern for allowed inputs.